lightning/src/routing/scoring.rs

```diff
@@ -1051,8 +1051,10 @@ fn success_probability(
 			// the amount, divided by the same integral from the minimum to the maximum liquidity
 			// bounds.
 			//
-			// Because the integral is simply a constant times (x - 0.5)^3, this means we simply
-			// subtract the two bounds mius 0.5 to the 3rd power.
+			// Because the integral from x to y is simply (y - 0.5)^3 - (x - 0.5)^3, we can
```

Comment:
Isn't the integral technically divided by 3? But then dividing the numerator by denominator eliminates it?